platforms/web/test-e2e/tests/payments/subscription_test.ts

```diff
@@ -41,111 +40,13 @@ function runTestSuite(props: ProviderProps, providerName: string) {
-    I.see('You have no subscription. Complete your subscription to start watching all movies and series.');
-    I.see('Complete subscription');
-
-    I.see('Payment method');
-    I.see('No payment methods');
-
-    I.see('Billing history');
-    I.see('No transactions');
-  });
-
-  Scenario(`I can see offered subscriptions - ${providerName}`, async ({ I }) => {
-    paidLoginContext = await I.registerOrLogin(paidLoginContext);
-
-    I.amOnPage(constants.paymentsUrl);
-
-    I.click('Complete subscription');
-    I.see('Choose plan');
-    I.see('Watch this on JW OTT Web App');
-
-    await within(props.monthlyOffer.label, () => {
-      I.see('Monthly');
-      I.see('First month free');
-      I.see('Cancel anytime');
-      I.see('Watch on all devices');
-      I.see(props.monthlyOffer.price);
-      I.see('/month');
-    });
-
-    await within(props.yearlyOffer.label, () => {
-      I.see('Cancel anytime');
-      I.see('Watch on all devices');
-      I.see(props.yearlyOffer.price);
-      I.see('/year');
-    });
-
-    I.see('Continue');
-  });
 
-  Scenario(`I can choose an offer - ${providerName}`, async ({ I }) => {
-    paidLoginContext = await I.registerOrLogin(paidLoginContext);
-
-    I.amOnPage(constants.offersUrl);
-
-    I.click(props.monthlyOffer.label);
-    I.seeCssPropertiesOnElements(props.monthlyOffer.label, { color: '#000000' });
-    I.seeCssPropertiesOnElements(props.yearlyOffer.label, { color: '#ffffff' });
-
-    I.click(props.yearlyOffer.label);
-    I.seeCssPropertiesOnElements(props.monthlyOffer.label, { color: '#ffffff' });
-    I.seeCssPropertiesOnElements(props.yearlyOffer.label, { color: '#000000' });
-
-    I.click('Continue');
-    I.waitForLoaderDone();
-
-    I.see('Yearly subscription');
-    I.see(props.yearlyOffer.price);
-    I.see('/year');
-
-    I.see('Redeem coupon');
-    I.see(props.yearlyOffer.price);
-    I.dontSee('Payment method fee');
-    I.dontSee(props.yearlyOffer.paymentFee);
-    I.see('Total');
-    if (props.applicableTax !== 0) {
-      I.see('Applicable tax (21%)');
-    }
-    I.clickCloseButton();
-  });
-
-  Scenario(`I can see payment types - ${providerName}`, async ({ I }) => {
-    paidLoginContext = await I.registerOrLogin(paidLoginContext);
-
-    await goToCheckout(I);
-
-    I.waitForText('Credit card');
-    I.waitForText('PayPal');
-    I.waitForText(props.paymentFields.creditCardholder);
-    I.waitForText('Card number');
-    I.waitForText('Expiry date');
-    I.waitForText('Security code');
-    I.waitForText('Continue');
-    I.dontSee("Clicking 'continue' will bring you to the PayPal site.");
-
-    I.click('PayPal');
-
-    I.waitForText("Clicking 'continue' will bring you to the PayPal site.");
-    I.dontSee('Card number');
-    I.dontSee('Expiry date');
-    I.dontSee('Security code');
-    I.waitForText('Continue');
+    I.useConfig(props.config);
   });
 
   Scenario(`I can open the PayPal site - ${providerName}`, async ({ I }) => {
```

Comment:
Looks like it is about payment now right? We check open in the "payments" suite.

Reply:
@ChristiaanScheermeijer do you think we can clean up some scenarios?

Reply:
Yes, I forgot to remove this one! I think this is the only duplicate scenario

Reply:
PR is updated. I removed the same scenarios in both files after splitting it into two files 😅

Reply:
"I can see my payments data", "I can choose an offer", "I can see offered subscriptions", and "I can see payment types" are still duplicated. Is it expected?

Reply:
The duplicated scenarios are removed now... this time for real!